tests/rhoai/scripts/gpu-setup.sh

```
@@ -0,0 +1,208 @@
+oc get nodes -o custom-columns=NAME:.metadata.name,INSTANCE:.metadata.labels.node\\.kubernetes\\.io/instance-type,STATUS:.status.conditions[-1].type
+
+# Debug: Check for GPU instance types and taints
+echo ""
+echo "--> DEBUG: Checking for GPU nodes and taints..."
+gpu_nodes=$(oc get nodes -o jsonpath='{range .items[*]}{.metadata.name}{","}{.metadata.labels.node\.kubernetes\.io/instance-type}{"\n"}{end}' | grep -E "g4dn|p3|p4|g5" | cut -d',' -f1 || echo "")
+
+if [ -n "$gpu_nodes" ]; then
+  echo "    Found GPU instance types:"
+  for node in $gpu_nodes; do
+    echo "    Node: $node"
+    echo "    Instance Type: $(oc get node $node -o jsonpath='{.metadata.labels.node\.kubernetes\.io/instance-type}')"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)